styles/style.css

```diff
@@ -120,18 +171,50 @@ p {
 /* hero */
 
 .hero {
-  background-image: url("header-bike.jpg");
+  background-image: url("../images/header-bike.jpg");
   background-color: var(--grey-light);
```

Comment:
here you can try `background-size: 75%` and it should look more similar to the final design